backend/src/users/model/user.interface.ts

```diff
@@ -0,0 +1,3 @@
+import { User } from "./user.entity";
+
+export interface IUser extends User {}
```

Comment:
Why not just use User?

Reply:
| think, to leave it empty now, for use later, but it's unnecessary in that case. 
I will remove it, and recreate as a other interface later if necessary :)